akvo/rsr/models/project.py

```diff
@@ -92,27 +92,54 @@ class Project(TimestampsMixin, models.Model):
     )
 
     STATUSES_COLORS = {
-        STATUS_NONE: 'black',
-        STATUS_NEEDS_FUNDING: 'orange',
-        STATUS_ACTIVE: '#AFF167',
-        STATUS_COMPLETE: 'grey',
-        STATUS_CANCELLED: 'red',
-        STATUS_ARCHIVED: 'grey',
+        '0': 'grey',
+        '1': 'orange',
+        '2': '#AFF167',
+        '3': 'grey',
+        '4': 'grey',
+        '5': 'red',
+        '6': 'grey',
     }
 
+    CODE_TO_STATUS = {
+        '0': 'N',
+        '1': 'H',
+        '2': 'A',
+        '3': 'C',
+        '4': 'C',
+        '5': 'L',
+        '6': 'R'
+    }
+
+    STATUS_TO_CODE = {
+        'N': '0',
+        'H': '1',
+        'A': '2',
+        'C': '3',
+        'L': '5',
+        'R': '6'
+    }
+
+    # Status combinations used in conditionals
+    EDIT_DISABLED = ['3', '5']
+    DONATE_DISABLED = ['0', '3', '4', '5', '6']
+    NOT_SUSPENDED = ['1', '2', '3', '4', '5']
+
     title = ValidXMLCharField(_(u'project title'), max_length=200, db_index=True, blank=True)
     subtitle = ValidXMLCharField(_(u'project subtitle'), max_length=200, blank=True)
-    status = ValidXMLCharField(
-        _(u'status'), max_length=1, choices=STATUSES, db_index=True, default=STATUS_NONE,
-        help_text=_(u'There are five different project statuses:<br/>'
-                    u'1) Needs funding: this project still needs funding and implementation has '
-                    u'not yet started.<br/>'
-                    u'2) Active: the implementation phase has begun.<br/>'
-                    u'3) Completed: the project has been completed.<br/>'
-                    u'4) Cancelled: the project never took place or work stopped before it was '
-                    u'fully implemented.<br/>'
-                    u'5) Archived: projects are archived when the reporting partner no longer uses '
-                    u'RSR.')
+    status = ValidXMLCharField(_(u'status'), max_length=1, choices=STATUSES, db_index=True, default=STATUS_NONE)
+    iati_status = ValidXMLCharField(
+        _(u'iati status'), max_length=1, choices=([('0', '')] + codelist_choices(ACTIVITY_STATUS)),
```

Comment:
Might be less confusing for users to name this field the same (u'status' instead of u'iati status'). In the project editor they'll otherwise see an IATI reference whilst that is not really necessary. Also update this in the migration btw.